generator/internal/language/rust_test.go

```diff
@@ -1107,6 +1107,7 @@ func TestRust_FormatDocComments(t *testing.T) {
 The next line has some extra trailing whitespace:` + "   " + `
```

Comment:
We need a new test for the link definitions.

Reply:
Done, added testing for link definitions.